pyspeckit/spectrum/fitters.py

```diff
@@ -1448,7 +1416,7 @@ def moments(self, fittype=None, **kwargs):
             The registered fit type to use for moment computation
         """
         fittype = fittype or self.fittype
-        return self.Registry.singlefitters[fittype].moments(
+        return self.Registry.multifitters[fittype].moments(
```

Comment:
On line 694 above, we need to remove singlefitters again.

We need to add a call to `Spectrum.baseline` before the first call to the fitter here.